includes/_footer.htm

```diff
@@ -32,7 +32,8 @@
 				'streams',
 				'quickstart',
 				'toc',
-				'upgrade'
+				'upgrade',
+				'content'
```

Comment:
I found this name a bit misleading.. could we rename it to `streams-content` or `streams-subpages`?